distributed/tests/test_worker.py

```diff
@@ -507,37 +507,6 @@ async def test_io_loop(s):
         assert w.io_loop is s.loop
 
 
-@gen_cluster(client=True, nthreads=[])
-async def test_spill_to_disk(c, s):
-    np = pytest.importorskip("numpy")
-    w = await Worker(
-        s.address,
-        loop=s.loop,
-        memory_limit=1200 / 0.6,
-        memory_pause_fraction=None,
-        memory_spill_fraction=None,
-    )
```

Comment:
This would hit the spill threshold after 1200/0.6*0.7=1400 **bytes** worth of total process memory, and the pause threshold after 1200/0.6*0.8=1600 bytes.

In other words, any futures at all managed to complete just thanks to a race condition with memory_monitor().
It was also a race condition to decide what would spill any of the keys, the zict LRU machinery used by target or the explicit evict() used by by spill.